smokey/src/main/java/frc/robot/commands/AutoBalanceStepCommand.java

```diff
@@ -11,8 +11,6 @@
 
 /**
  * Implements a command to perform a single step of an auto balancing routine. 
```

Comment:
So the preceding comment might need to be changed as well.

Reply:
Maybe something like: "Implements a command that will repeatedly iterate small drive steps and measurements to determine if the robot has obtained level, therefore attaining auto balance on the ramp."